src/components/Book.js

```diff
@@ -9,22 +9,29 @@ const Book = (props) => {
     <div className="book">
       <div className="title">
         <article>
```

Comment:
- [ ] Please update your Book title `font-size` and `font-family`, the author name `color`, category `color` and `font-weight` according to zeplin design 

Your design :point_down: 

![image](https://user-images.githubusercontent.com/75126481/150507906-3fd6a080-65b0-4b0f-826c-ca4bfba33aa5.png)

Zeplin design :point_down: 

![image](https://user-images.githubusercontent.com/75126481/150508260-20612861-5d3a-473c-b98d-936b7c1e709e.png)